src/transformStateWithClones.js

```diff
@@ -64,7 +64,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const clone = Object.assign({}, state);
+  const history = [];
+
+  for (let i = 0; i < transforms.length; i++) {
+    switch (transforms[i]['operation']) {
```

Comment:
```suggestion
    const { operation, properties } = transforms[i];
    switch (operation) {
```
теперь тебе будут доступны operation и properties конкретного transform[i]